src/app/crawlstart/crawlstart.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, OnInit } from '@angular/core';
+import {CrawlstartService} from "../crawlstart.service";
+
+@Component({
+  selector: 'app-crawlstart',
+  templateUrl: './crawlstart.component.html',
+  styleUrls: ['./crawlstart.component.css']
+})
+export class CrawlstartComponent implements OnInit {
+  crawlvalues: any = {
+    "crawlingMode": "url",
+    "crawlingURL": "",
+    "sitemapURL": "",
+    "crawlingFile": "",
+    "crawlingDepth": 3,
+    "crawlingDepthExtension": "",
+    "range": "domain",
+    "mustmatch": ".*",
+    "mustnotmatch": "",
+    "ipMustmatch": ".*",
+    "ipMustnotmatch": "",
+    "indexmustmatch": ".*",
+    "indexmustnotmatch": "",
+    "deleteold": "off",
+    "deleteIfOlderNumber": 0,
+    "deleteIfOlderUnit": "day",
+    "recrawl": "nodoubles",
+    "reloadIfOlderNumber": 0,
+    "reloadIfOlderUnit": "day",
+    "crawlingDomMaxCheck": "off",
+    "crawlingDomMaxPages": 1000,
+    "crawlingQ": "off",
+    "directDocByURL": "off",
+    "storeHTCache": "off",
+    "cachePolicy": "if fresh",
+    "indexText": "on",
+    "indexMedia": "off",
+    "xsstopw": "off",
+    "collection": "user",
```

Comment:
we should actually shouldn't be directly referencing the default values here, we have to receive them from server side but right now http://yacygrid.com:8300/yacy/grid/crawler/defaultValues.json?callback=p is down.